samples/scanner/scanner_conf.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Google Inc.
```

Comment:
This should go away yeah? I was thinking we'd have one sample and one default one to use

`samples/conf/forseti_conf.yaml.sample` and `conf/forseti_conf.yaml`